src/utilities/geometry/Test/Intersection_GTest.cpp

```diff
@@ -2349,33 +2349,39 @@ TEST_F(GeometryFixture, Offset) {
   poly1.push_back(Point3d(8, 10, 0));
   poly1.push_back(Point3d(11, 10, 0));
   poly1.push_back(Point3d(11, 7, 0));
+  Point3dVector poly2;
+  poly2.push_back(Point3d(7.5, 10.5, 0));
+  poly2.push_back(Point3d(11.5, 10.5, 0));
+  poly2.push_back(Point3d(11.5, 6.5, 0));
+  poly2.push_back(Point3d(7.5, 6.5, 0));
```

Comment:
An FYI but going forward you can do:

```
Point3dVector poly2 {
        {7.5, 10.5, 0.0},
        {11.5, 10.5, 0.0},
        {11.5, 6.5, 0.0},
        {7.5, 6.5, 0.0},
    };
```

This will avoid costly vector resize operations. (granted, this is a test here...).
The copy constructor will be called 7 times in the case of 4 push_backs, versus 4 times with initializer list.

The most efficient is this but it's annoying to type in a test

```
Point3dVector poly2;
poly2.reserve(4);
poly2.emplace_back(7.5, 10.5, 0);
poly2.emplace_back(11.5, 10.5, 0);
poly2.emplace_back(11.5, 6.5, 0);
poly2.emplace_back(7.5, 6.5, 0);
```

here's a testbed if you want to see for yourself: https://compiler-explorer.com/z/zf9EEr31j

Reply:
I'll use the initializer list - it's also closer to how it would be constructed in C# so I like that.